checkout/checkout_service.bal

```diff
@@ -110,17 +119,11 @@ service "CheckoutService" on new grpc:Listener(9094) {
             items: orderItems
         };
 
-        error? err = self.sendConfirmationMail(request.email, 'order);
-        if err is error {
-            log:printWarn(string `failed to send order confirmation to ${request.email}`, 'error = err);
-        } else {
-            log:printInfo(string `order confirmation email sent to ${request.email}`);
-        }
-
+        self.sendConfirmationMail(request.email, 'order);
```

Comment:
https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/src/checkoutservice/main.go#L277-L281

Reply:
Original example doesn't return the error to the client, instead logs. Since this is sending confirmation mail. it is okay to log